web/template/team.gotmpl

```diff
@@ -28,22 +28,40 @@
   </div>
 
   {{ if .Team.Members }}
-    <table class="govuk-table">
-      <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th scope="col" class="govuk-table__header">Name</th>
-          <th scope="col" class="govuk-table__header">Email</th>
-        </tr>
-      </thead>
-      <tbody class="govuk-table__body">
-        {{ range .Team.Members }}
+    <form action="{{ prefix (printf "/teams/remove-member/%d" .Team.ID) }}" method="POST">
+      <button type="submit" class="govuk-button govuk-button--secondary">
+        Remove selected from team
+      </button>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
           <tr class="govuk-table__row">
-            <td class="govuk-table__cell">{{ .DisplayName }}</td>
-            <td class="govuk-table__cell">{{ .Email }}</td>
+            <th scope="col" class="govuk-table__header"><span class="govuk-visually-hidden">Select</span></th>
+            <th scope="col" class="govuk-table__header">Name</th>
+            <th scope="col" class="govuk-table__header">Email</th>
+            <th scope="col" class="govuk-table__header"><span class="govuk-visually-hidden">Actions</span></th>
```

Comment:
I guess you don't need this?

Reply:
It's consistent with the users and add team member pages. I _think_ it's helpful to assistive tech for every column to have a header, but not certain.

Reply:
but there are only 3 columns?